src/include/stir/numerics/FastErf.h

```diff
@@ -53,11 +53,10 @@ class FastErf
   std::vector<double> erf_values_vec;
 
 public:
-  FastErf(){}
-
-  explicit FastErf(int num_samples)
+  explicit FastErf(const int num_samples = 1000, const float maximum_sample_value = 5)
```

Comment:
This doesn't work, as `maximum_sample_value` defaults to `this->maximum_sample_value`, not the argument name... It'd be better if `this->` was required (like `self.` in Python), but ...

Recommended way to initialise in a constructor is like this
```c++
explicit FastErf(const int num_samples = 1000, const float maximum_sample_value = 5)
 : _num_samples(num_samples), maximum_sample_value(maximum_sample_value )
{}
```
where there can be no conflict in naming... (it also avoids calling a default constructor for the members, but in this case that doesn't matter).

Of course, best to be uniform in naming and stick a `_` in front of all private members.

Reply:
and you should then remove the initialisation of the members, and corresponding doxygen. Otherwise we'll got an inconsistency at some point.